tox.ini

```diff
@@ -42,12 +43,6 @@ deps =
        -rrequirements/requirements-testing.txt
        -rrequirements/requirements-documentation.txt
 
-[testenv:py38-djangomain]
-ignore_outcome = true
-
-[testenv:py39-djangomain]
-ignore_outcome = true
```

Comment:
We don't test Django main against 3.8 or 3.9 so this is unused

Reply:
right